hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -2033,6 +2057,24 @@ func (r *HostedClusterReconciler) reconcileControlPlaneOperator(ctx context.Cont
 		return fmt.Errorf("failed to reconcile controlplane operator rolebinding: %w", err)
 	}
 
+	// Reconcile operator cluster role
+	controlPlaneOperatorClusterRole := controlplaneoperator.OperatorClusterRole(controlPlaneNamespace.Name)
```

Comment:
I wonder how we can avoid this, maybe passing this info through an annotation in the HCP from the HC? cc @sjenning thoughts?